addons/point_of_sale/static/src/app/printer/printer_service.js

```diff
@@ -17,8 +17,8 @@ export class PrinterService {
     setPrinter(newDevice) {
         this.device = newDevice;
     }
-    printWeb(el) {
-        this.renderer.whenMounted({ el, callback: window.print });
+    printWeb() {
```

Comment:
and this works in this situation too? 
![image](https://github.com/odoo/odoo/assets/118446179/17a1923d-3fcc-48e1-93cf-b61544622d4a)

Reply:
in restaurant it doesn't work for the tip receipt 
![image](https://github.com/odoo/odoo/assets/118446179/443f0032-08e6-48fb-a142-a7b5da8f4c03)

Reply:
because the receipt is not in the dom

Reply:
@vlst-odoo yeh, I made an ugly fix, by hiding the original receipt in the dom, I don't know if there is a better way

Reply:
i see, we can proceed with this and we'll come back later to refactor such that maybe we don't use `window.print` at all